src/test/java/space/obminyashka/items_exchange/util/AdvertisementDtoCreatingUtil.java

```diff
@@ -44,6 +49,32 @@ public static AdvertisementDto createExistAdvertisementDtoForUpdateWithUpdatedLo
                 CHANGEDLOCATION, AgeRange.OLDER_THAN_14, Season.SUMMER, Gender.FEMALE, "50", 2L, Collections.singletonList(JPEG));
     }
 
+    public static AdvertisementDto createExistAdvertisementDtoForUpdateWithNotValidSize() {
+        return getBuild(1L, "topic", "description", "shoes", true, DealType.EXCHANGE,
+                KHARKIV, AgeRange.OLDER_THAN_14, Season.SUMMER, Gender.MALE, "", 1L, Arrays.asList(JPEG, PNG));
+    }
+
+    public static AdvertisementDto createExistAdvertisementDtoForUpdateWithNotValidLocation() {
+        return getBuild(1L, "topic", "description", "shoes", true, DealType.EXCHANGE,
+                NOT_VALID_LOCATION, AgeRange.OLDER_THAN_14, Season.SUMMER, Gender.MALE, "40", 1L, Arrays.asList(JPEG, PNG));
+    }
+
+    public static AdvertisementDto createExistAdvertisementDtoForUpdateWithNotValidTopic() {
+        return getBuild(1L, "to", "description", "shoes", true, DealType.EXCHANGE,
+                KHARKIV, AgeRange.OLDER_THAN_14, Season.SUMMER, Gender.MALE, "40", 1L, Arrays.asList(JPEG, PNG));
+    }
+
+    public static AdvertisementDto createExistAdvertisementDtoForUpdateWithNotValidDescription() {
+        return getBuild(1L, "topic", NOT_VALID_DESCRIPTION, "shoes", true, DealType.EXCHANGE,
+                KHARKIV, AgeRange.OLDER_THAN_14, Season.SUMMER, Gender.MALE, "40", 1L, Arrays.asList(JPEG, PNG));
+    }
+
+
+    public static AdvertisementDto createExistAdvertisementDtoForUpdateWithNotValidWishesToExchange() {
+        return getBuild(1L, "topic", "description", NOT_VALID_WISHES, true, DealType.EXCHANGE,
+                KHARKIV, AgeRange.OLDER_THAN_14, Season.SUMMER, Gender.MALE, "40", 1L, Arrays.asList(JPEG, PNG));
```

Comment:
I see the same date 5 times here except one field. Just create default value for it and override if was get new value.

Reply:
ok